src/applets/quicknote/QuickNoteApplet.plugin.in

```diff
@@ -0,0 +1,8 @@
+[Plugin]
+Module=quicknoteapplet.so
+Name=QuickNote
+_Description=Quick & Easy Notes
+Authors=Jacob Vlijm, Budgie Desktop Developers
+Copyright=© 2017-2018 Ubuntu Budgie Developers, 2018 Budgie Desktop Developers
+Website=https://ubuntubudgie.org, https://solus-project.com
```

Comment:
Actually I'm going to be proactively changing Website= to https://budgie-desktop.org, please do the same.